### DISCUSSION TITLE
Google DMCA takedown?

### DISCUSSION THREAD

**Discussion by yodaluca23** [arichornlover/uYouEnhanced#431] [General]
I received this email today, I am unsure, if it is a phishing or not (it came from support@githubsupport.com) but I figured either way it would be best to discuss it here. If it's real then we need to figure that out, and if it's fake, then we can warn others...

![image](https://github.com/arichornlover/uYouEnhanced/assets/67206487/889796bd-bb81-4d01-828c-eb5382ef8c54)

The linked file:

https://enterprise.githubsupport.com/attachments/token/1u4kyYJnjA8HZTPMXOGBhRk4Q/?name=2024-03-06-google.rtf

It has basically every tweaked YouTube app listed, but I can't find it on GitHub's official DMCA repo, so I don't know...

https://github.com/github/dmca

**Comment by yodaluca23** [comment]
@arichornlover Did you receive a similar email?

**Reply by ego-lay-atman-bay** [reply]
According to [this](https://docs.github.com/en/support/learning-about-github-support/about-github-support)

> Email communication from GitHub Support will always be sent from either a `github.com` or `githubsupport.com` address.

So the email seems to be legit.

<details><summary>edit</summary>
<p>

Woops, meant to reply to the original message.

</p>
</details>

**Comment by yodaluca23** [comment]
Found this:

https://www.reddit.com/r/jailbreak/s/zrsZdswqC6

**Comment by arichornlover** [comment]
It’s legit. This is gonna be unfortunate but I got no choice. I will have to listen to them. I’m sorry guys.

**Reply by yodaluca23** [reply]
> It’s legit. This is gonna be unfortunate but I got no choice. I will have to listen to them. I’m sorry guys.

You can still update it and just make like a guide for noobs on how to use the GitHub actions to make their own... and then in bug reports just have them put the commit they're on instead of version.

**Reply by arichornlover** [reply]
> You can still update it and just make like a guide for noobs on how to use the GitHub actions to make their own... and then in bug reports just have them put the commit they're on instead of version.

I will update the guide.

Also I can’t remove the IPAs one by one. It’s too slow. This is not looking good to me.

**Reply by yodaluca23** [reply]
> I will update the guide.
> 
> 
> 
> Also I can’t remove the IPAs one by one. It’s too slow. This is not looking good to me. 

For me, they only sent about the two latest ones not any of the ones previous to it… Was it not the same for you?

**Reply by arichornlover** [reply]
I don’t think it’s just that. I think they are referring to all of the IPAs.

**Reply by yodaluca23** [reply]
> I don’t think it’s just that. I think they are referring to all of the IPAs.

I'll email them and let you know.

**Reply by arichornlover** [reply]
I already erased every single release here.

**Comment by arichornlover** [comment]
I have listened to the DMCA. However, I will still build uYouEnhanced with my IPA but I will not release it. I recommend anyone following the wiki page.

Note: I have removed all decrypted IPAs from the Wiki, I am sorry.

**Comment by mustafacakirogludev** [comment]
hey @arichornlover, since you are not sharing ipa files anymore due to the DMCA, I just compiled the new version for my personal use. After we download the compiled ipa file, do we need to do anything else other than deleting the newly created release and deleting the artifact to avoid a DMCA? Can you maybe add a sentence or two about this to the wiki so other people can avoid DMCAs from the ipas they created on their fork as well? Thanks!

**Reply by arichornlover** [reply]
I updated the wiki as you requested. Is this good? Also be aware of grammar errors. It was a little rushed at the bottom of the page.
https://github.com/arichornlover/uYouEnhanced/wiki/Building

**Reply by mustafacakirogludev** [reply]
I'd also add a sentence to the 3rd bullet point, something like "Don't forget to remove the IPA you built from your repo after you're done using it as sharing them are prohibited and might get your account suspended." to make sure people sees it, I feel like not many people will read the whole thing and close the wiki after the 3rd step. 

Thanks for your time! Really appreciate your work.

**Reply by yodaluca23** [reply]
> hey @arichornlover, since you are not sharing ipa files anymore due to the DMCA, I just compiled the new version for my personal use. After we download the compiled ipa file, do we need to do anything else other than deleting the newly created release and deleting the artifact to avoid a DMCA? Can you maybe add a sentence or two about this to the wiki so other people can avoid DMCAs from the ipas they created on their fork as well? Thanks!

If you just keep the release as a draft, it shouldn't matter.

**Comment by sfksuperman** [comment]
Anyone please a tutorial video how to make the ipa file. Thanks in advance.

**Reply by mustafacakirogludev** [reply]
Just follow the steps from the wiki? I never did it before but it was easy enough when I tried it yesterday.

https://github.com/arichornlover/uYouEnhanced/wiki/Building

**Reply by sfksuperman** [reply]
Where do I find YouTube ipa? And how to do "**Fork this repository**"?

**Reply by mustafacakirogludev** [reply]
> Where do I find YouTube ipa? 

I can't help with that since its prohibited but you can find it if you google decrypted youtube ipa and visit couple of websites, I did the same.

> And how to do "**Fork this repository**"?

Open the main uYouEnchanced repo, on the top right you have watch, fork and star buttons:

![Screenshot 2024-03-09 224206](https://github.com/arichornlover/uYouEnhanced/assets/101709759/79b1e68c-5527-4983-8a60-a48e05408ca7)

**Comment by kaosregulator** [comment]
is there a way to use working copy or hit hub app to do this in the iphone without a PC?

**Comment by arichornlover** [comment]
For anyone that saw the uYouEnhanced release `19.17.2-3.0.3`
if you saw the release and the ipa, I just wanted to let anyone know that it was a complete accident.
when I was compiling uYouEnhanced to make sure there were no compiling errors, it didn’t take too long for me to realize it auto-published a uYouEnhanced release.
it was up for at least 30+ minutes but than I took it down as soon as I checked the releases. I just hope this repo is fine.
